packages/client/src/index.tsx

```diff
@@ -25,6 +25,5 @@ function Root() {
   )
 }
 
-ReactDOM.render(<Root />, document.getElementById('root'))
```

Comment:
this takes over the browser and overwrites everything backstage puts there. we will need to separate the component exports from rendering the app